### PR TITLE
[🐴] New chat dialog refresh

### PR DESCRIPTION
**Pull request by estrattonbailey**
- [x] Android border radius on dialog

### CODE REVIEW

File: src/components/Dialog/index.tsx [hunk 1]
```diff
@@ -276,6 +277,8 @@ export const InnerFlatList = React.forwardRef<
         a.h_full,
         {
           marginTop: 40,
+          borderTopLeftRadius: 40,
+          borderTopRightRadius: 40,
```
Comment (estrattonbailey) [inline src/components/Dialog/index.tsx:281]: Prevents overflow in the corners, needs to match `Inner`

File: src/components/Dialog/index.web.tsx [hunk 2]
```diff
@@ -199,18 +201,18 @@ export const ScrollableInner = Inner
 
 export const InnerFlatList = React.forwardRef<
   FlatList,
-  FlatListProps<any> & {label: string}
->(function InnerFlatList({label, style, ...props}, ref) {
+  FlatListProps<any> & {label: string} & { webInnerStyle?: StyleProp<ViewStyle> }
+>(function InnerFlatList({label, style, webInnerStyle, ...props}, ref) {
```
Comment (estrattonbailey) [inline src/components/Dialog/index.web.tsx:205]: Need to override some styles on web bc of this composition

File: src/state/queries/actor-autocomplete.ts [hunk 3]
```diff
@@ -20,6 +20,7 @@ export const RQKEY = (prefix: string) => [RQKEY_ROOT, prefix]
 export function useActorAutocompleteQuery(
   prefix: string,
   maintainData?: boolean,
+  limit?: number,
```
Comment (estrattonbailey) [inline src/state/queries/actor-autocomplete.ts:23]: Really shouldn't use positional props but didn't want to make edits in all the other files

File: src/components/dms/NewChatDialog/index.tsx [hunk 4]
```diff
@@ -0,0 +1,496 @@
...
+          paddingHorizontal: 0,
+          marginTop: 0,
+          paddingTop: 0,
+        }),
+      ]}
+      webInnerStyle={[a.py_0, {maxWidth: 500, minWidth: 200}]}
+      onScrollBeginDrag={() => Keyboard.dismiss()}
+    />
+  )
+}
```
Comment (haileyok) [inline src/components/dms/NewChatDialog/index.tsx:496]: holy shit almost 500LOC what a screen!

File: src/state/queries/actor-autocomplete.ts (hunk 3 above)
Comment (haileyok) [reply src/state/queries/actor-autocomplete.ts:23]: yea all good, felt the same way when i added the `maintainData`

**Review by haileyok — APPROVED** [review]
amazing work on this! works great. made a small adjustment to the prop so the dismiss runs sync.

**Pull request merged**